.github/workflows/regenerate-protobuf.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       - name: Install grpc and wsrpc
         if: github.actor != 'app-token-issuer-engops[bot]' && steps.cache-deps.outputs.cache-hit != 'true'
         run: |
-          go install google.golang.org/protobuf/cmd/protoc-gen-go@${{ steps.tool-versions.outputs.protoc-gen-go_version }}
+          go install google.golang.org/protobuf/cmd/protoc-gen-go@v${{ steps.tool-versions.outputs.protoc-gen-go_version }}
```

Comment:
I added the `v` prefix to the `protoc-gen-go` version, otherwise the GHA failed.

Reply:
Was it failing silently? Did this change trigger all the other changes? b/c I don't see an explicit version bump anywhere else.

Reply:
Yes, once I made this change it triggered all the other changes. Otherwise it failed: https://github.com/smartcontractkit/chainlink-protos/actions/runs/15213678718/job/42793891820#step:12:16